src/browser/FileProxy.js

```diff
@@ -81,13 +77,10 @@
 
         var unicodeLastChar = 65535;
 
-    /** * Exported functionality ***/
+        /** * Exported functionality ***/
 
         exports.requestFileSystem = function (successCallback, errorCallback, args) {
             var type = args[0];
-            // Size is ignored since IDB filesystem size depends
-            // on browser implementation and can't be set up by user
-            var size = args[1]; // eslint-disable-line no-unused-vars
```

Comment:
No reason to allocate memory if the variable is not used.

Reply:
remove it.

Reply:
The variable? Already did. I just wanted to highlight this change, because the diff is pretty big and lots for "formatting".